config/image.style.news_image_500_300_.yml

```diff
@@ -0,0 +1,15 @@
+uuid: 1df4e5d2-40a5-44e1-852e-f6d9bca285f9
+langcode: en
+status: true
+dependencies: {  }
+name: news_image_500_300_
```

Comment:
Change the image style, there shouldn't be any extra `_` at the end of a machine name.

Reply:
Changed machine name